corehq/apps/export/models/new.py

```diff
@@ -211,6 +232,14 @@ def generate_instance_from_schema(schema, domain, app_id=None):
         return instance
 
 
+class CaseExportInstance(ExportInstance):
+    case_type = StringProperty()
+
+
+class FormExportInstance(ExportInstance):
+    xmlns = StringProperty()
+
+
```

Comment:
yeah i've been back and forth on this. on one hand it's nice to have `ExportInstance` be agnostic to case/form, though i can see where we may need this down the road. i really do want to avoid have 3 classes for every class in the ExportInstance tree (Base, Case, and Form)